### PR TITLE
[Docs] update Japanese translation of feature_led_matrix.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/feature_led_matrix.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_led_matrix.md [hunk 1]
```diff
@@ -1,96 +1,357 @@
...
+  // 物理位置に対する LED インデックス
+  { 188,  16 }, { 187,  48 }, { 149,  64 }, { 112,  64 }, {  37,  48 }, {  38,  16 }
+}, {
+  // フラグへの LED インデックス
+  1, 4, 4, 4, 4, 1
+} };
+```
 
-## カスタムレイヤー効果
+最初の部分、`// LED インデックスへのキーマトリックス`は、キーの電気的なマトリックス 行&列を使ってこの LED が表すキーをシステムに伝えます。 2番目の部分、`// 物理位置に対する LED インデックス`はキーボード上の LED の物理的な `{ x, y }` 位置を表します。`{ x, y }` のデフォルトの予期される値の範囲は `{ 0..224, 0..64 }` の両端を含む範囲です。このデフォルトの予想範囲はアニメーションのキーボードの中心を計算する効果によるものです。これらの位置を計算する最も簡単な方法は、キーボードが格子であり、キーボードの左上が `{ x, y }` 座標 `{ 0, 0 }` を表し、キーボードの右下が `{ 224, 64 }` を表すと想像することです。これを基礎として、物理位置を計算するために以下の式を使うことができます:
```
Comment (s-show) [inline docs/ja/feature_led_matrix.md:95]: 1段落に複数の文があるので、主語の後に句読点があった方が読みやすいかなと思います。
```suggestion
最初の部分、`// LED インデックスへのキーマトリックス`は、キーの電気的なマトリックス 行&列を使ってこの LED が表すキーをシステムに伝えます。 2番目の部分、`// 物理位置に対する LED インデックス`は、キーボード上の LED の物理的な `{ x, y }` 位置を表します。`{ x, y }` のデフォルトの予期される値の範囲は、 `{ 0..224, 0..64 }` の両端を含む範囲です。このデフォルトの予想範囲は、アニメーションのキーボードの中心を計算する効果によるものです。これらの位置を計算する最も簡単な方法は、キーボードが格子であり、キーボードの左上が `{ x, y }` 座標 `{ 0, 0 }` を表し、キーボードの右下が `{ 224, 64 }` を表すと想像することです。これを基礎として、物理位置を計算するために以下の式を使うことができます:
```

File: docs/ja/feature_led_matrix.md [hunk 2]
```diff
@@ -1,96 +1,357 @@
...
+x = 224 / (NUMBER_OF_COLS - 1) * COL_POSITION
+y =  64 / (NUMBER_OF_ROWS - 1) * ROW_POSITION
+```
 
-    void led_matrix_indicators_kb(void) {
-    led_matrix_set_value(index, value);
-    }
+ここで、NUMBER_OF_COLS、NUMBER_OF_ROWS、COL_POSITION & ROW_POSITION は全て、電気的レイアウトではなく、キーボードの物理的レイアウトに基づいています。
+
+前述のように、キーボードの中心はデフォルトで `{ 112, 32 }` であることを期待されますが、LED の物理的な `{ x, y }` 位置をより正確に計算したい場合は、これを変更することができます。キーボードの設計者は、キーボードの新しい中心を使って、あるいは `{ x, y }` 値のより多くの可能性を許可して、config.h の中で `#define LED_MATRIX_CENTER { 112, 32 }` を実装することができます。x または y の最大値は 255 で、推奨される最大は 244 であることに注意してください。これにより、アニメーションがリセットされる前に余裕が生まれます。
```
Comment (s-show) [inline docs/ja/feature_led_matrix.md:104]: 「possibility」には「可能性」の他に「候補」という訳もあるようですので、次のような訳はどうでしょうか？
あと、タイポがありました。
```suggestion
前述のように、キーボードの中心はデフォルトで `{ 112, 32 }` であることを期待されますが、LED の物理的な `{ x, y }` 位置をより正確に計算したい場合は、これを変更することができます。キーボードの設計者は、キーボードの新しい中心を使って、あるいは `{ x, y }` 値のより多くの候補を許可して、config.h の中で `#define LED_MATRIX_CENTER { 112, 32 }` を実装することができます。x または y の最大値は 255 で、推奨される最大は 224 であることに注意してください。これにより、アニメーションがリセットされる前に余裕が生まれます。
```

File: docs/ja/feature_led_matrix.md [hunk 3]
```diff
@@ -1,96 +1,357 @@
...
+`led_matrix_kb.inc` はキーボードディレクトリのルートにある必要があります。
+
+独自の効果をコードで使うには、`LED_MATRIX_EFFECT()` で指定した効果名の前に `LED_MATRIX_CUSTOM_` を追加するだけです。例えば、`LED_MATRIX_EFFECT(my_cool_effect)` として宣言された効果は、以下のように参照されます:
+
+```c
+led_matrix_mode(led_MATRIX_CUSTOM_my_cool_effect);
+```
+
+```c
+// !!!DO NOT ADD #pragma once !!!//
```
Comment (s-show) [inline docs/ja/feature_led_matrix.md:209]: 翻訳もれです
```suggestion
// !!!#pragma once を追加しないこと!!!//
```

File: docs/ja/feature_led_matrix.md [hunk 4]
```diff
@@ -1,96 +1,357 @@
...
+
+```c
+#define LED_MATRIX_KEYPRESSES // キー押下に反応
+#define LED_MATRIX_KEYRELEASES // キーリリースに反応(キー押下の代わり)
+#define LED_MATRIX_FRAMEBUFFER_EFFECTS // フレームバッファ効果を有効化
+#define LED_DISABLE_TIMEOUT 0 // led が自動的にオフになるまでのミリ秒数
+#define LED_DISABLE_AFTER_TIMEOUT 0 // 非推奨: 効果を無効にするまでのティック数
+#define LED_DISABLE_WHEN_USB_SUSPENDED // 一時停止時に効果をオフ
+#define LED_MATRIX_LED_PROCESS_LIMIT (DRIVER_LED_TOTAL + 4) / 5 // タスクの実行ごとにアニメーションで処理する LED の数を制限(キーボードの応答性を高める)
+#define LED_MATRIX_LED_FLUSH_LIMIT 16 // アニメーションが LED を更新する頻度を制限するミリ秒16 (16ms) は、60fps に制限することと同じ (キーボードの応答性が向上します)
```
Comment (s-show) [inline docs/ja/feature_led_matrix.md:272]: 「increases keyboard responsiveness」という一文は直前にも登場していますので、訳は揃えたほうが良いと思います。
あと、句読点の「。」が抜けています。
```suggestion
#define LED_MATRIX_LED_FLUSH_LIMIT 16 // アニメーションが LED を更新する頻度を制限するミリ秒。16 (16ms) は、60fps に制限することと同じ (キーボードの応答性を高める)
```

**Review by peepeetee — APPROVED** [review]

**Comment by zvecr** [comment]
Closing as this would have gone stale multiple times by now.

**Pull request closed without merging**